server/data_scanpy/scanpy_adaptor.py

```diff
@@ -0,0 +1,305 @@
+import warnings
```

Comment:
I realize this is a very late-breaking comment...

However, we actually don't use ScanPy.  We use Anndata which is a separate module, incorporated by ScanPy, but really a sibling.   For legacy reasons, we called it the "scanpy" module.  

Ideally, we would rename this to "anndata_adaptor", and excise the use of the term "scanpy" from the code base.

Reply:
Done